internal/proxy/task_delete.go

```diff
@@ -252,10 +255,16 @@ func (dt *deleteTask) Execute(ctx context.Context) (err error) {
 		ts := dt.deleteMsg.Timestamps[index]
 		_, ok := result[key]
 		if !ok {
+			msgid, err := dt.idAllocator.AllocOne()
+			if err != nil {
+				return merr.WrapErrIDAllocatorFailed("Fail to alloc msgid when Execute in delete")
```

Comment:
The original error lost, just `errors.Wrap(err, "failed to allocate MsgID for delete")`